subscribie/themes/theme-builder/builder/layout.html

```diff
@@ -114,6 +114,9 @@
                             <li class="nav-item w-auto">
                                 <a class="nav-link text-white" href="/shop-owner-login">Login</a>
                             </li>
```

Comment:
@elliottmotson please may you do instead

```
                            <li class="nav-item w-auto">
                                <a class="nav-link text-white" href="/shop-owner-login">Login</a> / <a class="nav-link text-white" href="/start-building">Register</a>
                            </li>
```

See: https://github.com/Subscribie/subscribie/issues/778#issue-1139813411